systemd/systemd.go

```diff
@@ -312,15 +323,39 @@ type reporter interface {
 	Notify(string)
 }
 
+// SNAP_BOOTIMG_PART_NUM  is the number of available boot image partitions
+const MIN_UBUNTU_RELEASE = 18
+
+// Based on the release info, determine if systemd can be clever about
+// things like daemon-reload
+func smartSystemd() bool {
```

Comment:
Could we base this decision on systemd.Version() instead so that other distros can benefit from this as well?

Reply:
That was initial proposal when we discuss this some time ago. But I was told one should not rely simply on systemd version. But in principle I do agree with you, systemd version should be main indicator.